2.2.js

```diff
@@ -0,0 +1,16 @@
+num = 1;
+for (i = 0; i < 100; i++) {

```

Comment:
da sie napisac fizzbuzza uzywajac 2ch ifow (kazdy else to 1 if) - sprobuj to tak napisac, wiedzac ze:
+ console.log zawsze najpierw wykonuje to co ma w srodku a potem dopiero zwraca wynik
+ || (operator logiczny OR) zwraca pierwsza wartosc jezeli jest prawdziwa lub druga wartosc gdy pierwsza jest falszywa.
+ przyda sie zmienna pomocnicza i konkatenacja stringow

zeby oddac Ci honor - Twoje rozwiazanie jest szybsze. ale jesli mialabys nie zrobic FizzBuzz z 3 i 5 a np. FizzBuzzCosTamCostamIJeszczeCos z 3,5,7,13,17,23,etc. to Twoj kod mialby milion linii ;)